vector3d.h

```diff
@@ -0,0 +1,83 @@
+//3d vector class for Rendera, Yashas M Salian
+
+#include<math.h>
```

Comment:
All the `.h` header files are only in C++ because of backward compatibility with C. When writing C++, we prefer the headers with a prefix of `c`. I.e., we want to include `cmath` in this case and not `math.h`.